sqlmesh/core/scheduler.py

```diff
@@ -277,8 +299,21 @@ def run(
             end_bounded=end_bounded,
             selected_snapshots=selected_snapshots,
         )
+
         if not merged_intervals:
-            return True
+            next_ready_interval_start = get_next_model_interval_start(self.snapshots.values())
+
+            time_format_str = "%Y-%m-%d %-I:%M%p %Z"  # Example: 2024-01-01 6:00AM UTC
+            utc_time = next_ready_interval_start.strftime(time_format_str)
+            local_timezone = datetime.now().astimezone().tzinfo
+            local_time = next_ready_interval_start.astimezone(local_timezone).strftime(
```

Comment:
This looks like something that belongs in `utils/date.py`. I'm sure there will be more places where you need this.